### PR TITLE
Update V3-Software_Platform_Requirements.md

### PR DESCRIPTION
**Pull request by cetome**
Adding requirement 3.4.12 for secure update and encrypted communication channel: Issue #74

### CODE REVIEW

**Comment by scriptingxss** [comment]
Thanks for the PR. Let's go with the first commit 7baf4a8095aefa4a6d6b6a271b2690bd6e08dabd

**Review by scriptingxss — CHANGES_REQUESTED** [review]
Thanks for the PR. Let's go with the first commit 7baf4a8

File: en/V3-Software_Platform_Requirements.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The Linux operating system is one of the most popular in IoT. It has many featur
 | **3.4.9** | Verify that encrypted firmware images are securely decrypted on the device. |  | ✓ | ✓ |
 | **3.4.10** | Verify that the device authenticates to the update server component prior to downloading the update.| ✓ | ✓ | ✓ |
 | **3.4.11** | Verify that firmware updates are stored encrypted server-side. | | ✓ | ✓ |
+| **3.4.12** | Verify that software and firmware updates are transmitted using an encrypted communication channel. This is to protect the update process against man-in-the-middle attacks such as firmware recovery, injection of compromised update. | ✓ | ✓ | ✓ |
```
Comment (scriptingxss) [inline en/V3-Software_Platform_Requirements.md:68]: Could we remove the second sentence? 

`This is to protect the update process against man-in-the-middle attacks such as firmware recovery, injection of compromised update. `

File: en/V1-IoT_Ecosystem_Requirements.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ To ensure the security of all software produced, the build process for the syste
 | -- | ---------------------- | - | - | - |
 | **1.2.1** | Verify that each application (including firmware) in the ecosystem maintains a software bill of materials (SBOM) cataloging third-party components, versioning, and published vulnerabilities. | ✓ | ✓ | ✓ |
 | **1.2.2** | Verify that potential areas of risk that come with the use of third-party and open-source software have been identified and that actions to mitigate such risks have been taken. | ✓ | ✓ | ✓ |
-| **1.2.3** | Verify the device is released with firmware and configuration appropriate for a release build (as opposed to debug versions). | ✓ | ✓ | ✓ |
+| **1.2.3** | Verify the device is released with firmware configured with secure defaults appropriate for a release build (as opposed to debug versions). | ✓ | ✓ | ✓ |
```
Comment (scriptingxss) [inline en/V1-IoT_Ecosystem_Requirements.md:32]: Thanks for the PR here. We are mixing Chapter 1 and chapter 3 commits but it's not a big deal.

Comment (cetome) [reply en/V1-IoT_Ecosystem_Requirements.md:32]: Definitely not what I was trying to do oops 😞 
I'm going to update it and push a new PR.
I am a huge noob in GitHub and I don't know how to remove some files from the PR.

**Review by cetome — COMMENTED** [review]
Removed the changes.

**Pull request merged**